release-notes/latest.md

```diff
@@ -1,3 +1,7 @@
 #### <sub><sup><a name="5842" href="#5842">:link:</a></sup></sub> feature
 
 * Added recover for panic error that used to crash the cluster. Now it should be less easy to panic (we hope!) and if it does, panic error could be found on Stderr and log. #5842
+
+#### <sub><sup><a name="5861" href="#5861">:link:</a></sup></sub> feature
+
+* @evanchaoli Make `--config` flag of `fly set-pipelines` command to support `-` for reading pipeline config from stdin. #5861
```

Comment:
I think this also affects other fly commands that use `PathFlag`:
* `format-pipeline`
* `validate-pipeline`
* `execute` (the task config, plus vars-from-file)
*  `login` (the CACert - this one's a bit obscure though)

Reply:
Yes, exactly. But I only intended to modify `set-pipeline`, others are just side-effects. Maybe we can just consider others as a hide feature.